### PR TITLE
Feature/change user type masthead

### PR DESCRIPTION
**Pull request by chsomani**
Changing the type of the user element in the masthead.

### CODE REVIEW

File: lib/components/Masthead/Masthead.tsx [hunk 1]
```diff
@@ -67,7 +66,7 @@ export interface MastheadProperties {
         attr?: InlinePopup.Attributes;
     };
     toolbarItems?: Array<MastheadToolbarItem>;
-    user?: MastheadUserItem;
+    user?: JSX.Element;
```
Comment (Coord) [inline lib/components/Masthead/Masthead.tsx]: nit: try React.ReactNode

File: lib/components/Masthead/Masthead.tsx [hunk 2]
```diff
@@ -2,7 +2,6 @@ import * as React from 'react';
 import * as classnames from 'classnames/bind';
 import { MethodNode } from '../../Common';
 import * as InlinePopup from '../InlinePopup';
```
Comment (Coord) [inline lib/components/Masthead/Masthead.tsx]: nit: clean up unused dependency too

File: lib/components/Masthead/Masthead.tsx (hunk 1 above)
Comment (chsomani) [reply lib/components/Masthead/Masthead.tsx]: updated type to React.ReactNode

**Comment by chsomani** [comment]
Closing the PR after resolving issues reported!

**Pull request merged**